paper.md

```diff
@@ -33,48 +33,62 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
 
 In broad-brush, pedagogically-focused workshops and summer schools tend to follow a classic academic model where novices learn a skillset from experts, while project-focused workshops focus on people collaboratively exercising the skillset they already have.
 A disadvantage of the summer school model is that it can tends to focus on a one-way flow of information from instructor to student, and can discount the potential contributions by students.
-A disadvantage of the hackathon model is the common perception (whether accu rate or not) that the week is designed for experts in technical tools, which may discourage others from attending.
-In 2014, we started AstroHackWeek to try to fill the gaps between these models. The HackWeek model combines pedagogy (often focused on statistical and computational techniques) with room for collaborative hacks or creative projects, with the goal of encouraging collaboration and learning among people at various stages of their career.
+A disadvantage of the hackathon model is the common perception (whether accurate or not) that the week is designed for experts in technical tools, which may discourage others from attending.
+In 2014, we started Astrohack week to try to fill the gaps between these models. The hack week model combines pedagogy (often focused on statistical and computational techniques) with room for collaborative hacks or creative projects, with the goal of encouraging collaboration and learning among people at various stages of their career.
 
 ![alt text](fig/HackSpectrum.png "Comparison of Extracurricular Workshop Models")
 
-As of the publication of this paper, we have run five such HackWeek events: three focused on Astronomy, one focused on Neuroscience, and one focused on Geophysics.
-Below we will share some of the philosophy behind the HackWeek model, practical lessons we have learned in organizing these events, and recommendations for future HackWeeks in other disciplines.
+As of the publication of this paper, we have run five such hack week events: three focused on Astronomy, one focused on Neuroscience, and one focused on Geophysics.
```

Comment:
Fingers crossed it won't take that long. 

In other news: I've been co-organizing Python in Astronomy this year, which shares many features with the hack week model, including tutorials, unconference sessions, sprints and a hack day. I've referred to it a couple of times in the text, but I'm curious if anyone minds to include it more formally (e.g. in the figure)?